modules/contact-form/grunion-contact-form.php

```diff
@@ -2038,13 +2038,39 @@ function process_submission() {
 		 * with this work around. */
 		add_filter( 'wp_insert_post_data', array( $plugin, 'insert_feedback_filter' ), 10, 2 );
 
+		$all_values_string = '';
+		
+		// Avoid warnings when print_r() has been disabled on hosting provider for security reasons
+		// function_exists will return false for functions disabled with the disable_functions ini directive
+		if ( function_exists( 'print_r' ) ) {
```

Comment:
Since the filter allows you to customize the output and since `@` gets rid of the warning,  you don't need the function checks anymore. You can keep it all on one line, allowing anyone to change `@print_r( $all_values, true ` into whatever they want on their own site.